### PR TITLE
Passcards: The Instrument of Bureaucracy

### PR DESCRIPTION
**Pull request by crunchypretzels**
Adds a passcard selection to the loadout for humans(and xenos living in human space).
Replaces the Sol passport with a new sprite, by me, because the Bay one was a bit wrong.
Fixes the Penny plushie.
Moves the paper passport selection and renames some backends of them. Check your loadouts.
image
[Himeo, Gadpathur, Coalition, Vysokan,
Elyran(animated), Eridani(animated), Biesel, Dominia
Silversun(Idris Only), Plutonian, Solarian, Assunzionii.]

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

File: code/modules/clothing/under/accessories/badges.dm [hunk 1]
```diff
@@ -244,42 +244,236 @@
...
+	contained_sprite = 1
+	slot_flags = null
+	w_class = ITEMSIZE_TINY
+	flippable = FALSE
+	v_flippable = FALSE
+
+/obj/item/clothing/accessory/badge/passcard_dominia
+	name = "dominian passcard"
+	desc = "A passcard issued to citizens of the Empire of Dominia."
+	desc_fluff = "Dominian passcards, aside from the usual information, also include details such as ones House, their remaining Mo'ri'zal, their status as an Edict Breaker, or- most uniquely- their most recent \
```
Comment (listerla) [inline code/modules/clothing/under/accessories/badges.dm:342]: Should be "one's house" rather than "ones house" I imagine.

Comment (crunchypretzels) [reply code/modules/clothing/under/accessories/badges.dm]: got you gamer 😎

**Review by Geevies — CHANGES_REQUESTED** [review]
all of the passports and passcards should really be subtypes of one main thing, to prevent continuously having to redefine every icon and weight class and flippablity and etc.

File: code/modules/clothing/under/accessories/badges.dm [hunk 2]
```diff
@@ -244,42 +244,226 @@
...
+/obj/item/clothing/accessory/badge/passcard/burszia
+	name = "burszian passcard"
+	desc = "A passcard issued to Burszian Hephaestus employees and- owned IPCs- working abroad."
+	desc_fluff = "Despite protest from the Himean representatives in government, Hephaestus Industries- citing their 'Home is where the Hephaestus is' initiative- is permitted to issue up to five thousand \
+	sponsored passcards to participating employees on a yearly basis, both to remind them of their home and to save on imported labor costs."
+	icon_state = "passcard_burs"
+	item_state = "passcard_burs"
+
+/obj/item/clothing/accessory/badge/passcard/konyang
+	name = "burszian passcard"
```
Comment (listerla) [inline code/modules/clothing/under/accessories/badges.dm:425]: Named this burszian, when it's konyanger.

File: code/modules/clothing/under/accessories/badges.dm [hunk 3]
```diff
@@ -244,42 +244,226 @@
...
+	name = "republic of biesel passcard"
+	desc = "A passcard issued to citizens of the Republic of Biesel, typically from planets in Biesel proper and smaller territories."
+	desc_fluff = "A passcard is a modern evolution of the state-issued identification card, with all the functionality of a driver's license, birth certificate, passport, or other document, \
+	updated as necessary or able by a central government. The concept was pioneered in the early days of the Sol Alliance, and continues in most human stellar nations to this day, owing to the availability \
+	and price of consumer plastics and self-powered microholograms."
+	icon = 'icons/clothing/accessories/passcards.dmi'
+	icon_state = "passcard_ceti"
+	item_state = "passcard_ceti"
+	overlay_state = "passport_ceti"
+	contained_sprite = 1
```
Comment (Geevies) [inline code/modules/clothing/under/accessories/badges.dm:257]:
```suggestion
	contained_sprite = TRUE
```

File: code/modules/clothing/under/accessories/badges.dm [hunk 4]
```diff
@@ -244,42 +244,226 @@
...
+
+//passports
+
+/obj/item/clothing/accessory/badge/passport
+	name = "biesellite passport"
+	desc = "A passport issued to a citizen of the Republic of Biesel."
+	icon = 'icons/clothing/accessories/passcards.dmi'
+	icon_state = "passport_ceti"
+	item_state = "passport_ceti"
+	contained_sprite = 1
```
Comment (Geevies) [inline code/modules/clothing/under/accessories/badges.dm:440]:
```suggestion
	contained_sprite = TRUE
```

File: code/modules/clothing/under/accessories/badges.dm [hunk 5]
```diff
@@ -244,42 +244,226 @@
...
+	icon_state = "passport_coc"
+	item_state = "passport_coc"
+
+/obj/item/clothing/accessory/badge/passport/elyra
+	name = "elyran passport"
+	desc = "A passport issued to a citizen of the Serene Republic of Elyra. Vintage!"
+	icon_state = "passport_elyra"
+	item_state = "passport_elyra"
+
+
```
Comment (Geevies) [inline code/modules/clothing/under/accessories/badges.dm:464]:
```suggestion
```

**Review by Geevies — APPROVED** [review]
very cute, good work

**Review by Doxxmedearly — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**